inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/eltwise_b_fs_yx_fsv16.cl

```diff
@@ -58,22 +66,37 @@ KERNEL(eltwise_b_fs_yx_fsv16)(INPUTS_DECLS
 
     DO_ELTWISE
 
+#if HAS_FUSED_OPS
+    FUSED_OPS;
+#if BLOCK_SIZE != 1
+    OUTPUT_TYPE_BLOCK out = TO_OUTPUT_TYPE_BLOCK(FUSED_OPS_RESULT);
+#else
+    OUTPUT_TYPE out = TO_OUTPUT_TYPE(FUSED_OPS_RESULT);
+#endif
+#else
+#if BLOCK_SIZE != 1
+    OUTPUT_TYPE_BLOCK out = TO_OUTPUT_TYPE_BLOCK(res);
+#else
+    OUTPUT_TYPE out = TO_OUTPUT_TYPE(res);
+#endif
+#endif
+
 #ifdef LEFTOVERS
     if ((f_block + 1) * FEATURE_SLICE_SIZE > OUTPUT_FEATURE_NUM) {
         const uint sglid = get_sub_group_local_id();
         if (sglid < OUTPUT_FEATURE_NUM % FEATURE_SLICE_SIZE) {
             for (uint block_x = 0; block_x < BLOCK_SIZE; block_x++) {
 #if BLOCK_SIZE != 1
-                output[output_offset + block_x * output_x_pitch + sglid] = ACTIVATION_TYPED(res[block_x], ACTIVATION_PARAMS_TYPED);
+                output[output_offset + block_x * output_x_pitch + sglid] = ACTIVATION_TYPED(out[block_x], ACTIVATION_PARAMS_TYPED);
 #else
-                output[output_offset + block_x * output_x_pitch + sglid] = ACTIVATION_TYPED(res, ACTIVATION_PARAMS_TYPED);
+                output[output_offset + block_x * output_x_pitch + sglid] = ACTIVATION_TYPED(out, ACTIVATION_PARAMS_TYPED);
```

Comment:
ACTIVATION or ACTIVATION_TYPED shouldn't be used together with FUSED_OPS, please move to #else of #if HAS_FUSED_OPS